Automattic/woocommerce-payments
# Fix manual capture status source

Fixes https://github.com/Automattic/woocommerce-payments/issues/9551

#### Changes proposed in this Pull Request
The manual capture feature flag was removed some time ago, and the code no longer relies on this flag. This PR updates the WooPayments settings class to retrieve the manual capture status from the same option used by other classes.


#### Testing instructions

Confirm that
* When `Issue an authorization on checkout, and capture later` checkbox is checked, `WooCommerce` -> `Status` shows `Auth and Capture` as `Enabled`
* and `Disabled` with unchecked checkbox 
* no failures/errors when opening the WooCommerce status page

-------------------


**Post merge**
